tests/bootstrap.php

```diff
@@ -21,5 +21,5 @@
  *
  */
 
-require_once __DIR__ . '/../../../tests/bootstrap.php';
+require_once __DIR__ . '/bootstrap.php';
```

Comment:
the file requires itself :see_no_evil: 

use ``require_once __DIR__.'/../../../lib/base.php';``

Reply:
OMG. Yes. 🤣